schedule/ha/bv/ha_cluster_node.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: ha_cluster_node
+description: >
+  HA Cluster Test. Schedule for all nodes.
+
+  Some settings are required in the job group or test suite for this schedule to work.
+
+  The other settings required in the job group are.
+
+  CLUSTER_NAME must be defined for all jobs as a string.
+  HA_CLUSTER_INIT must be defined to a non false value in the parent job
+  HA_CLUSTER_JOIN must be defined for the rest of the jobs, and it must contain the
+  hostname of the parent job (the job where HA_CLUSTER_SETUP is defined to init)
+  HOSTNAME must be defined to different hostnames for each node.
+  MAX_JOB_TIME is recommended to be defined as well to a high value (ex. 20000)
+  All jobs with the exception of the parent job must include a PARALLEL_WITH setting
+  referencing the parent job.
+  SLE_PRODUCT must be defined and set accordingly.
+  And of course, YAML_SCHEDULE must point to this file.
+vars:
+  BOOT_HDD_IMAGE: '1'
+  USE_SUPPORT_SERVER: '1'
+  HDD_SCC_REGISTERED: '1'
+  VIRTIO_CONSOLE: '0'
+  HA_CLUSTER: '1'
+  # Below have to be entered in the OpenQA UI because it doesn't read this YAML
```

Comment:
Suggestion:
```
# Below setting must be defined in the openQA UI because macros for %VERSION%,
# %ARCH% and %BUILD% are usually not defined yet when this file is being loaded
```